include/mc_status.h

```diff
@@ -0,0 +1,25 @@
+#ifndef __MC_STATUS_H__
+#define __MC_STATUS_H__
+
+#include "common.h"
+
+typedef enum
+{
+    MC_STATUS_OK                = 0,
+    MC_STATUS_OVERCURRENT       = 1,
+    MC_STATUS_NOT_SET           = 2,
+    MC_STATUS_HOLE_SENSOR_ERROR = 3,

```

Comment:
```suggestion
    MC_STATUS_HALL_SENSOR_ERROR = 3,
```

Reply:
It literally says hole sensor in the manual

Reply:
![MicrosoftTeams-image](https://github.com/badgerloop-software/sc1-mcc/assets/72111610/b1fb1bc4-c632-4f8e-924f-aed6d2e027a0)

Reply:
Mitsuba is stupid